internal/cmd/controller/agent/agent.go

```diff
@@ -79,18 +79,19 @@ func AgentWithConfig(ctx context.Context, agentNamespace, controllerNamespace, a
 	return objs, err
 }
 
-// agentBootstrapSecret creates the fleet-agent-bootstrap secret
+// agentBootstrapSecret creates the fleet-agent-bootstrap secret from the
+// import-token-local secret and adds the APIServer options.
```

Comment:
Does this hold for any downstream cluster as well? If so:
```suggestion
// import-token-<clusterName> secret and adds the APIServer options.
```